src/libcadet/AutoDiff.hpp

```diff
@@ -24,7 +24,7 @@
 
 #if defined(ACTIVE_SFAD) || defined(ACTIVE_SETFAD)
 
-	#define SFAD_DEFAULT_DIR 80
+	#define SFAD_DEFAULT_DIR 3300
```

Comment:
This should probably not be merged into `master`, right? @sleweke

Reply:
change was reverted